### DISCUSSION TITLE
Invert a scope

### DISCUSSION THREAD

**Discussion by Julian** [community/community#9864] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there syntax for inverting a scope? It seems like possibly not, but such a thing might be nice.

E.g. if I want to find uses of my library, I want to write something like `mylib ~user:julian` (or `^user:julian`, I don't remember what syntax lucene queries tend to use for this) to exclude my own repositories.

**Comment by intrigus-lgtm** [comment, marked as answer]
You should be able to change your search term to this:
`mylib AND NOT user:Julian`
Also see https://cs.github.com/about/syntax#boolean :)

(Example search without exclusion: ["https://twitter.com/JulianWasTaken"](https://cs.github.com/?scopeName=All+repos&scope=&q=%22https%3A%2F%2Ftwitter.com%2FJulianWasTaken%22)
with: ["https://twitter.com/JulianWasTaken" AND NOT user:Julian](https://cs.github.com/?scopeName=All+repos&scope=&q=%22https%3A%2F%2Ftwitter.com%2FJulianWasTaken%22+AND+NOT+user%3AJulian)
)

**Reply by Julian** [reply]
Ah, of course, thanks!
